materialize-firebolt/transactor.go

```diff
@@ -123,17 +110,20 @@ type transactor struct {
 }
 
 // firebolt is delta-update only, so loading of data from Firebolt is not necessary
-func (t *transactor) Load(it *pm.LoadIterator, _ <-chan struct{}, _ <-chan struct{}, loaded func(int, json.RawMessage) error) error {
-	panic("delta updates have no load phase")
+func (t *transactor) Load(it *pm.LoadIterator, loaded func(int, json.RawMessage) error) error {
+	for it.Next() {
```

Comment:
Yes, and the connector is responsible for driving the iterator to completion. However, it can expect that it will never yield any Load keys if it's delta-updates.